Kickstarter-iOS/Views/Controllers/RewardAddOnSelectionViewController.swift

```diff
@@ -94,18 +133,30 @@ final class RewardAddOnSelectionViewController: UITableViewController {
     self.pledgeShippingLocationViewController.view.rac.hidden
       = self.viewModel.outputs.shippingLocationViewIsHidden
 
+    self.viewModel.outputs.configureContinueCTAViewWithData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        self?.continueCTAView.configure(with: data)
+      }
+
     self.viewModel.outputs.configurePledgeShippingLocationViewControllerWithData
       .observeForUI()
       .observeValues { [weak self] data in
         self?.pledgeShippingLocationViewController.configureWith(value: data)
       }
 
-    self.viewModel.outputs.loadAddOnRewardsIntoDataSource
+    self.viewModel.outputs.loadAddOnRewardsIntoDataSourceAndReloadTableView
       .observeForUI()
       .observeValues { [weak self] rewards in
         self?.dataSource.load(rewards)
         self?.tableView.reloadData()
       }
+
+    self.viewModel.outputs.loadAddOnRewardsIntoDataSource
```

Comment:
Split these out into a signal that updates the data source _and_ reloads the table view, and another that only updates the data source. This is so that the table view is not reloaded each time we change a selection on one of the cells but also to maintain data integrity for when we recycle cells as we scroll the view.